code/game/objects/items/weapons/legcuffs.dm

```diff
@@ -166,6 +171,7 @@
 		feedback_add_details("handcuffs","B")
 		to_chat(C, "<span class='userdanger'>[src] ensnares you!</span>")
 		C.Weaken(weaken)
+		playsound(src.loc, 'sound/effects/snap.ogg', 50, TRUE)
```

Comment:
Implied src.

You can also check bola/energy in legcuffs.dm - It has a hitsound defined as hitsound = 'sound/weapons/tase.ogg'

You should set the hitsound for all bolas and then let energy bola override it, instead of adding a redundant line to play a special sound on impact. This will most likely result in two sounds being played at once when a e-bola impact.

Reply:
Implied src fixed. I have done as you asked, and set default hitsound for all bolas, unless children (like energy bola over here) overrides it. Tested it, and all works fine, no duplicit sound.